argocd/overlays/moc-infra/applications/envs/moc/smaug/kruize/kruize.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: argoproj.io/v1alpha1
 kind: Application
 metadata:
-  name: hpo
+  name: Kruize-HPO
```

Comment:
This can't contain capital letters:

https://kubernetes.io/docs/concepts/overview/working-with-objects/names/#dns-subdomain-names